robot_ws/src/voice_interaction_robot/src/voice_interaction_robot/test/integration_test.py

```diff
@@ -125,47 +123,63 @@ def run_test(self, test):
         retry_count = 0
         while retry_count <= self.max_retries_per_test:
             if retry_count > 0:
-                rospy.loginfo("Retrying failed test {}".format(test))
+                self.get_logger().info("Retrying failed test {}".format(test))
                 time.sleep(self.test_sleep_time)
             test.run_test()
             time.sleep(self.test_sleep_time)
-            if test.check_result(self.last_cmd_vel):
-                rospy.loginfo("test passed")
+            if test.check_result(self.get_logger(), self.last_cmd_vel):
+                self.get_logger().info("test passed")
                 return
             else:
                 retry_count += 1
 
-        rospy.loginfo("test failed")
+        self.get_logger().info("test failed")
         
     def wait_for_voice_interaction_nodes(self):
         required_nodes = set([
             '/lex_node', 
             '/voice_interaction', 
             '/voice_command_translator'
         ])
-        while not required_nodes.issubset(rosnode.get_node_names()):
-            time.sleep(0.1)
+        time.sleep(1)
+        # while not required_nodes.issubset(rosnode.get_node_names()):
+            # time.sleep(0.1)
```

Comment:
Why remove this? Theres `self.get_node_names()`
https://github.com/ros2/rclpy/blob/master/rclpy/rclpy/node.py#L1546